docusaurus.config.js

```diff
@@ -58,7 +58,8 @@ const config = {
     }
   ],
   title: 'Reown Docs',
-  tagline: 'Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure.',
+  tagline:
+    'Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure.',
```

Comment:
```suggestion
  tagline: 'Reown gives developers the tools to build user experiences that make digital ownership effortless, intuitive, and secure.',
```

Reverting it back to the previous version as I have a PR up in parallel which may end up in a conflict if this is merged.